docs/contracts.rst

```diff
@@ -1059,6 +1059,9 @@ derived contract. Arguments have to be given either in the
 inheritance list or in modifier-style in the derived constuctor.
 Specifying arguments in both places is an error.
 
+If the constructor of a base contract has no arguments, it will be implicitly
```

Comment:
This kind of implies that the point where the arguments to a base constructor are specified has an influence about the execution of the base constructor. Perhaps frame it differently to just state that arguments to base constructors that do not take parameters do not have to be specified, but that already sounds a little redundant.

Reply:
Yeah, if you put it like that it sounds obvious. But this was very unexpected to me, so I was trying to document it trying to make it clear for newbies in my same situation. I'm thinking how to rephrase it...

Reply:
What needs to be made clear is that a derived contract remains abstract if it doesn't specify the arguments to all of its base contracts' constructors. However, if the constructor of a base contract has no arguments, it cannot cause the derived contract to be abstract because all of its arguments are trivially specified.

Although the base constructor arguments are specified with function call syntax, they should not be confused for a function call.

Is there something like this anywhere in the docs?